.github/workflows/build-syft-image.yml

```diff
@@ -0,0 +1,51 @@
+name: Build syft image
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - syft.Dockerfile
+    
+  pull_request:
+    branches:
+      - main
+    paths:
+      - syft.Dockerfile
+
+
+env:
+  REGISTRY: quay.io/redhat-appstudio
+  IMAGE_NAME: syft
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Extract version from Dockerfile
+      id: extract-version
+      run: echo "::set-output name=version::$(grep -oP '(?<=ARG syft_version=")[^"]*' syft.Dockerfile)"
```

Comment:
Apparently this is deprecated https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

(see the [examples](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/#examples) for the new way to do things)